### DISCUSSION TITLE
How can i use the name field with "fields"?

### DISCUSSION THREAD

**Discussion by luvwinnie** [fastapi/fastapi#7807] [Questions] [answered]
### First check

* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

How can I use the model with name of fields??
Because of the specification of my software, i want to use the name of "fields".

Is it possible to se the name "fields" in BaseModel?

The below code i tried to defined it shows the errors.

```
class Model(BaseModel):
    name_1:str
    name_2:str
    fields: List[Dict]

model = Model()
```

```
Traceback (most recent call last):
  File "model.py", line 76, in <module>
    class Model(BaseModel):
  File "pydantic/main.py", line 191, in pydantic.main.ModelMetaclass.__new__
  File "pydantic/utils.py", line 93, in pydantic.utils.validate_field_name
NameError: Field name "fields" shadows a BaseModel attribute; use a different field name with "alias='fields'".
```

**Comment by phy25** [comment]
This is definitely an upstream issue: https://github.com/samuelcolvin/pydantic/issues/1250

**Comment by luvwinnie** [comment, marked as answer]
Thanks , i have been able to solve my problem from [samuelcolvin/pydantic#1250](https://github.com/samuelcolvin/pydantic/issues/1250) issue

**Comment by tiangolo** [comment]
Thanks for the help here @phy25 :bowing_man: 

Thanks for reporting back and closing the issue @luvwinnie :+1:
